iOSClient/Settings/NCSettings.m

```diff
@@ -114,7 +114,7 @@ - (void)initializeForm
 //    [section addFormRow:row];
     
     // EndToEnd Encryption
-    NSString *title = [NSString stringWithFormat:@"%@ (%@)",NSLocalizedString(@"_e2e_settings_", nil), NSLocalizedString(@"_experimental_", nil)];
+    NSString *title = [NSString stringWithFormat:@"%@",NSLocalizedString(@"_e2e_settings_", nil)];
```

Comment:
You can asign string directly NSLocalizedString(@"e2e_settings", nil)
No need to use stringwithformat function as we are using single string